libs/core/src/components/pds-box/pds-box.tsx

```diff
@@ -182,35 +182,102 @@ export class PdsBox {
   @Prop() shadow?: BoxShadowSizeType;
 
   /**
-   * Size of the column for all screen sizes that are not explicitly set.
+   * Sets the default column width for the component. This value applies from the smallest screen size (XS) upwards, unless overridden by a breakpoint-specific size prop at that breakpoint or larger.
    */
   @Prop() size?: BoxColumnType;
 
   /**
-   * At screen sizes less than the `XS` breakpoint, this will take the number of columns specified.
+   * At the XS breakpoint (typically the smallest screen sizes, 0px to 575px), this will take the number of columns specified. This overrides the base size prop for this specific range.
```

Comment:
Please make this specific change:
```suggestion
   * At the XS breakpoint, this prop will take the number of columns specified. This overrides the base size prop for this specific range.
```